config/settings.yml

```diff
@@ -3,11 +3,15 @@ dsauth:
 
 rails:
   secret_key_base: <%= ENV.fetch('SECRET_KEY_BASE') %>
+  force_ssl: <%= ENV.fetch("FORCE_SSL", true) %>
 
 devise:
   key: <%= ENV.fetch('DEVISE_KEY') %>
   salt: <%= ENV.fetch('DEVISE_SALT') %>
 
+doorkeeper:
+  force_ssl_in_redirect_uri: <%= ENV.fetch('FORCE_SSL_IN_REDIRECT_URI', false) %>
```

Comment:
Could we default to a secure state here - so by default we force SSL and we force SSL URIs?